doc/classes/CharacterBody2D.xml

```diff
@@ -179,6 +173,12 @@
 		<member name="moving_platform_wall_layers" type="int" setter="set_moving_platform_wall_layers" getter="get_moving_platform_wall_layers" default="0">
 			Collision layers that will be included for detecting wall bodies that will act as moving platforms to be followed by the [CharacterBody2D]. By default, all wall bodies are ignored.
 		</member>
+		<member name="safe_margin" type="float" setter="set_safe_margin" getter="get_safe_margin" default="0.08">
```

Comment:
Should it be named `collision_safe_margin` instead?

Reply:
I wondered too, but since the setter was `set_safe_margin`, and that's the only direct way this could be used until now, I opted for that one. If we prefer to use `collision_safe_margin`, then the setter and getter should be renamed too.

CC @godotengine/physics

Reply:
I find the "collision_safe_margin" (maybe biased with habit) seems a bit long, however, it is a parameter that rarely needs to be changed, being in a Collision group a bit apart seems relevant to me, so we can rename it for me (in his group).

Reply:
So you'd suggest renaming to `collision_safe_margin` and adding a "Collision" group? (And then also rename the setter/getter to `set/get_collision_safe_margin`)

Reply:
I went with an intermediate approach to keep the short `safe_margin` name (internally it's even just `margin`, and `safe_margin` is used as parameter name in `move_and_slide`), and just moved it back to the end in a prefix-less "Collision" group.

Reply:
I wasn't entirely convinced tbh, this morning I thought of `collision_margin` (safe is a bit useless), your solution avoid a breaking change.

Reply:
And btw, there are some pretty long names currently in the properties, in the "floor" or "moving_platform" group, if you have an opinion/suggestion (or reduz) on that.